bigquery_etl/operational_monitoring/__init__.py

```diff
@@ -72,14 +72,16 @@ def _query_up_to_date(dataset, slug, basename, project_last_modified):
 
 
 def _write_sql_for_data_type(
-    query, project, dataset, slug, render_kwargs, probes, data_type
+    query, project, dataset, om_project, render_kwargs, probes, data_type
 ):
-    normalized_slug = _bq_normalize_name(slug)
+    normalized_slug = _bq_normalize_name(om_project["slug"])
     render_kwargs.update(
         {
             "source": query["source"],
             "probes": _get_name_and_sql(query, probes, "probes", data_type),
-            "slug": slug,
+            "slug": om_project["slug"],
+            "channel": om_project["channel"],
+            "pref": om_project.get("boolean_pref"),
```

Comment:
I could be missing something, but I feel like it might make more logical sense to update the `om_projexct`-specific `render_kwargs` in the function calling this, rather than in this method.

Reply:
Good point! `slug` is here because the raw slug needs to be set right before writing the query sql file but the others don't need to be there.